testing/integration/tests/address_test.rs

```diff
@@ -11,8 +11,9 @@ use fvm_shared::state::StateTreeVersion;
 use fvm_shared::version::NetworkVersion;
 use num_traits::Zero;
 
-#[test]
-fn basic_address_tests() {
+// XXX RE-ENABLE THIS AFTER MERGE WITH BUILTIN ACTORS
+// #[test]
+fn _basic_address_tests() {
```

Comment:
`#[cfg(ignore)]`?

Reply:
then it makes the imports all mad so... eh

Reply:
Ah, ok, fine.